rest-api-spec/src/main/resources/rest-api-spec/api/cluster.get_decommission_awareness.json

```diff
@@ -0,0 +1,19 @@
+{
+  "cluster.get_decommission_awareness": {
+    "documentation": {
+      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/decommission/",
+      "description": "Get details and status of decommissioned attribute"
+    },
+    "stability": "experimental",
```

Comment:
Please confirm stability of this API.

Reply:
As discussed above, we will mark it stable once all meta issues are closed and merged. Have created an issue tracking that.
Meta issue list - https://github.com/opensearch-project/OpenSearch/issues/3957